addons/im_livechat/tests/test_get_operator.py

```diff
@@ -221,3 +222,48 @@ def test_priority_by_number_of_chat_all_operators_exceed_limit(self):
         self._create_chat(livechat_channel, second_operator)
         self._create_chat(livechat_channel, second_operator)
         self.assertEqual(first_operator, livechat_channel._get_operator())
+
+    def test_get_by_expertise(self):
+        dog_expert = self.env["im_livechat.expertise"].create({"name": "dog"})
+        cat_expert = self.env["im_livechat.expertise"].create({"name": "cat"})
+        dog_expert_operator = self._create_operator(expertise=dog_expert)
+        cat_expert_operator = self._create_operator(expertise=cat_expert)
+        livechat_channel = self.env["im_livechat.channel"].create(
+            {
+                "name": "Livechat Channel",
+                "user_ids": [dog_expert_operator.id, cat_expert_operator.id],
+            }
+        )
+        self.assertEqual(
+            dog_expert_operator, livechat_channel._get_operator(expertise_ids=dog_expert)
```

Comment:
Kwarg is actually named `expertises` (which is debatable btw as _ids suffix suggests a recordset which is the case here).